packages/wrangler/e2e/helpers/e2e-wrangler-test.ts

```diff
@@ -47,15 +47,21 @@ export class WranglerE2ETestHelper {
 
 	runLongLived(
 		wranglerCommand: string,
-		{ cwd = this.tmpPath, ...options }: WranglerCommandOptions = {}
+		{
+			cwd = this.tmpPath,
+			cleanup = true,
```

Comment:
Cleanup is a bit general. In my mind it makes me think it will be deleting files and stuff. How about `autoStop`, `stopOnTestFinished` or something like that?

Reply:
https://github.com/cloudflare/workers-sdk/pull/9327/commits/81b8737bdac9a0016e9d91b4e71a011755372b95